containerd/tests/test_containerd.py

```diff
@@ -0,0 +1,150 @@
+    aggregator.assert_metric(NAMESPACE + '.mem_usage_failcnt_total')
+    aggregator.assert_metric(NAMESPACE + '.mem_usage_limit')
+    aggregator.assert_metric(NAMESPACE + '.mem_usage_max')
+    aggregator.assert_metric(NAMESPACE + '.mem_usage')
+    aggregator.assert_metric(NAMESPACE + '.mem_writeback')
+    aggregator.assert_metric(NAMESPACE + '.per_cpu')
+    aggregator.assert_metric(NAMESPACE + '.pids_current')
+    aggregator.assert_metric(NAMESPACE + '.pids_limit')
+    aggregator.assert_metric(NAMESPACE + '.go.goroutines')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_alloc')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_alloc_bytes_total')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_buck_hash_sys')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_frees_total')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_gc_cpu_fraction')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_gc_sys')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_heap_alloc')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_heap_idle')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_heap_inuse')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_heap_objects')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_heap_released')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_heap_sys')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_last_gc_time_seconds')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_lookups_total')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_mallocs_total')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_mcache_inuse')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_mcache_sys')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_mspan_inuse')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_mspan_sys')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_next_gc')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_other_sys')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_stack_inuse')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_stack_sys')
+    aggregator.assert_metric(NAMESPACE + '.go.memstats_sys')
+    aggregator.assert_metric(NAMESPACE + '.go.threads')
+    aggregator.assert_metric(NAMESPACE + '.grpc.server_msg_received_total')
+    aggregator.assert_metric(NAMESPACE + '.grpc.server_handled_total')
+    aggregator.assert_metric(NAMESPACE + '.grpc.server_msg_sent_total')
+    aggregator.assert_metric(NAMESPACE + '.grpc.server_started_total')
+    aggregator.assert_metric(NAMESPACE + '.http.requests_total')
+    aggregator.assert_metric(NAMESPACE + '.process.cpu_seconds_total')
+    aggregator.assert_metric(NAMESPACE + '.process.max_fds')
+    aggregator.assert_metric(NAMESPACE + '.process.open_fds')
+    aggregator.assert_metric(NAMESPACE + '.process.resident_memory')
+    aggregator.assert_metric(NAMESPACE + '.process.start_time_seconds')
```

Comment:
you can finish with a `aggregator.assert_all_metrics_covered()` 💅

Reply:
A little nit picky, but if these are related metrics can we have them defined in another const file and just iterate through it to check that they exist? Quick example - https://github.com/DataDog/integrations-core/blob/master/process/tests/common.py#L7